v20.2/cockroach-debug-zip.md

```diff
@@ -39,7 +40,7 @@ The file produced by `cockroach debug zip` can contain highly sensitive, unanony
 
 There are two scenarios in which `debug zip` is useful:
 
-- To collect all of your nodes' logs, which you can then parse to locate issues. It's important to note, though, that `debug zip` can only access logs from active nodes. See more information [on this page](#collecting-log-files).
+- To collect all of your nodes' logs, which you can then parse to locate issues. It's important to note, though, that `debug zip` can only access logs from active nodes. See more information [below](#collecting-log-files).
```

Comment:
Nit but if we are going to edit this bit, suggested editing to the more descriptive "For more information, see [Collecting log files](#collecting-log-files) below.